namespaces/cloud-platform-live-0.k8s.integration.dsd.io/nick-sample-app-dev/resources/ecr.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=master"
+
+  team_name = "my-team-name"
+  repo_name = "my-app-name"
+}
+
+resource "kubernetes_secret" "ecr-repo" {
+  metadata {
+    name      = "ecr-repo-my-app-name"
+    namespace = "my-environment"
```

Comment:
This should match the namespace you want the values to be made accessible to `nick-sample-app-dev` in this case.